COLLABORATOR_GUIDE.md

```diff
@@ -227,3 +228,33 @@ messages. However, you are only allowed to force push to any Node.js
 branch within 10 minutes from your original push. If someone else
 pushes to the branch or the 10 minute period passes, consider the
 commit final.
+
+### Long Term Support
+
+#### What is LTS?
+
+Long Term Support (often referred to as *LTS*) guarantees application developers 
+a 30 month support cycle with specific versions of Node.js.
+
+You can find more information [in the full LTS plan](https://github.com/nodejs/lts#lts-plan).
+
+#### How does LTS work?
+
+Once a release enters LTS, no new features may be added to that release. Changes are limited to
+bug fixes, security updates, possible npm updates, documentation updates, and certain
+performance improvements that can be demonstrated to not break existing applications.
+Semver-minor changes are only permitted if required for bug fixes. Semver-major changes are only
+permitted if required for critical security and bug fixes.
+
+Once a release moves into Maintenance mode, only **critical** bugs, **critical** security fixes,
+and documentation updates will be permitted.
+
+#### How can I help?
+
+When you send your pull request, consider including information about
+whether your change is breaking. If you think your patch can be backported,
+please feel free to include that information in the PR thread.
+
+#### Who is doing the backporting?
+
+Currently all backporting is being done by [@nodejs/release](https://github.com/nodejs/node/blob/master/README.md#release-team).
```

Comment:
The current plan is for commits to cherry pick into a staging branch first (e.g. `v4.x-staging`), which can be done by anyone -- in fact, the preference would be for the person landing the commit in master to also land the commit in the staging branch around the same time if the commit needs to land in the LTS. Then, when the LTS WG determines that a new LTS release is required, selected commits will be picked from the staging branch to the release branch (from from `v4.x-staging` to `v4.x`. It would be the responsibility of the person preparing the commit to pull the commits out of the `v4.x-staging` branch. The LTS WG works with the Release team on this process.